src/Symfony/Component/HttpFoundation/Session/Attribute/NamespacedAttributeBag.php

```diff
@@ -43,7 +43,7 @@ public function __construct($storageKey = '_sf2_attributes', $namespaceCharacter
      */
     public function has($name)
     {
-        $attributes = $this->resolveAttributePath($name);
+        $attributes = & $this->resolveAttributePath($name);
```

Comment:
Please remove the space after the unary operator.
https://github.com/symfony/symfony/pull/14090 fixed it in the whole codebase recently but it looks like your PR was created based on an older version of the code